content/millennium/r4/entities/organization.md

```diff
@@ -64,6 +64,43 @@ Search for Organizations that meet supplied query parameters:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
```

Comment:
Looks like R4 is missing a documentation on OperationDefinition similar to DSTU2: https://fhir.cerner.com/millennium/dstu2/conformance/operation-definition/

We may have to add that in and add documentation about our new operation definition.

Reply:
Updated here, tried to be consistent with R4 StructureDefinition: 2854e95f8b9560f0ca35af72369b9adbda6b13c5